pyvista/core/dataset.py

```diff
@@ -411,7 +411,7 @@ def points(self, points: Union[VectorArray, NumericArray, _vtk.vtkPoints]):
             self.Modified()
             return
         # otherwise, wrap and use the array
-        points = _coerce_pointslike_arg(points)
+        points = _coerce_pointslike_arg(points, copy=False)
```

Comment:
I want to make sure this is explicit like usage elsewhere in this file

Reply:
This leaves this line as the only line without an explicit usage.  This one could also probably use `copy=False` too, which makes it even more apparent that we should use this as the default.

https://github.com/pyvista/pyvista/blob/c78438f89f5e3e905ed1c885d3d0cb2cd7917997/pyvista/utilities/geometric_objects.py#L490